mobile/src/main/java/org/openhab/habdroid/ui/PreferencesActivity.java

```diff
@@ -322,6 +344,25 @@ protected void updateAndInitPreferences() {
                 getParent(vibrationPref).removePreference(vibrationPref);
             }
 
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+                Log.d(TAG, "Removing alarm clock prefs");
+                getPreferenceScreen().removePreference(alarmClockPrefCat);
+            } else {
+                updateAlarmClockEnabledPreferenceIcon(alarmClockEnabledPref,
+                        getPreferenceBool(alarmClockEnabledPref, false));
+                alarmClockEnabledPref.setOnPreferenceChangeListener((preference, newValue) -> {
+                    updateAlarmClockEnabledPreferenceIcon(preference, newValue);
+                    return true;
+                });
+
+                setEditorSummary(alarmClockItemPref, getPreferenceString(alarmClockItemPref,
+                        ""));
```

Comment:
Nit: The line wrap is slightly weird, should wrap after `alarmClockItemPref,` for better readability.

Reply:
Will do